Templates/CreateLogin.sqltemplate

```diff
@@ -0,0 +1,27 @@
+/* Template Args:
+   1. Login Name
+   2. Database Name
+   3. Password
+ */
+USE [%[2]s]
+GO
+CREATE LOGIN [%[1]s] WITH PASSWORD=N'%[3]s', DEFAULT_DATABASE=[%[2]s], DEFAULT_LANGUAGE=[us_english], CHECK_EXPIRATION=OFF, CHECK_POLICY=OFF
+GO
+ALTER LOGIN [%[1]s] ENABLE
+GO
+EXEC sys.sp_addsrvrolemember @loginame = N'%[1]s', @rolename = N'sysadmin'
```

Comment:
I question whether this should give admin privs. If the intent is for the import tool to be able to create databases and such, it already has permission and such at that point via an existing user. 

Aside from that, I don't think there should be any reason to go out of our way to prepare an admin account. If we must, it should probably be separate from knight.